client/lib/analytics/ad-tracking.js

```diff
@@ -339,15 +335,13 @@ let isNanigansConfigured = false;
 function setupNanigansGlobal() {
 	isNanigansConfigured = true;
```

Comment:
Yeah, will do. 

This whole Nanigans setup is kind of weird, we should not need to rely on this variable at all actually and follow what the other trackers do, ie setup global vars on module init and do the actual initialization (if needed) at script loading time. While Nanigans uses this kind of "lazy" loading logic which overcomplicates things. I had started refactoring this one too but then left it as it for another time since it felt a bit tricky and the diff was already big enough.